objects-revisited/main.js

```diff
@@ -0,0 +1,146 @@
+// declaring a object called business
+const business = {
+  opens: '9am',
+  closes: '9pm',
+  totalEmployees: 3,
+  daysOpen: ['M', 'Tu', 'W', 'Th', 'F'],
+  Marke: {
+    position: 'Manger',
+    daysOfWeekWorking: ['M', 'Tu', 'F'],
+  },
+  Aiden: {
+    position: 'Front End Developer',
+    daysOfWeekWorking: ['W', 'Th', 'F'],
+  },
+  bob: {
+    position: 'Intern',
+    daysOfWeekWorking: ['Tu', 'W', 'Th', 'F'],
+  },
+};
+
+function addEmployees(obj) {
+  // crating a new instance of the 'XMLHttpRequest' object
+  // to make HTTP requests to a server
```

Comment:
I like that you comment your code. The best comments, though, are not ones that simply repeat what the code does (I can read the code for that and don't need the comment), but are ones that explain _why_ the code is written as it is. In other words, code comments should answer the questions "Why is this code here?" or "What is the purpose of this code?"